Source/Data/memo/falsecount.txt

```diff
@@ -1,5 +1,4 @@
 由中            is removed due to too many false positive count, we can add it
                 it back after modifying the original textpool files
 得與		is removed due to too many false positive count, we can add it
-                it back after modifying the original textpool files
```

Comment:
This line seems to be removed accidentally, am I right?

Reply:
Yes, sorry about that
I ran a script that removes duplicates